apollo-router/src/configuration/mod.rs

```diff
@@ -333,6 +334,7 @@ impl Server {
 /// Listening address.
 #[derive(Debug, Clone, PartialEq, Deserialize, Serialize, JsonSchema)]
 #[serde(untagged)]
+#[non_exhaustive]
 pub enum ListenAddr {
```

Comment:
cant quite think of anything else, but non exhaustive is the safe bet anyway 😅

Reply:
Yeah I’ve reverted this one. I was a bit too mechanical in adding it everywhere :)